src/electron/window-frame-updater.ts

```diff
@@ -7,15 +7,19 @@ import { WindowStateStoreData } from './flux/types/window-state-store-data';
 export class WindowFrameUpdater {
     constructor(private readonly windowStateStore: WindowStateStore, private readonly browserWindow: BrowserWindow) {}
 
-    public initialize(): void {
+    public async initialize(): Promise<void> {
```

Comment:
why is this `async` ??? am I missing something here? are the functions from `browserWindow` async functions ?